lib/assessment.js

```diff
@@ -39,9 +73,38 @@ var select = function select(array, callback, thisArg) {
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
+  var result = true;
+  array.forEach(function(currentValue, index, array) {
+    result = result && callback(currentValue, index, array);
+  });
 
+  return result;
```

Comment:
This is `every` again, and it's not necessarily helpful for completing this question. It's good that you made an effort, though. No credit.